plugins/a11y-keyboard/msd-a11y-keyboard-manager.c

```diff
@@ -49,9 +49,13 @@
 
 #include "mate-settings-profile.h"
 #include "msd-a11y-keyboard-manager.h"
+#ifdef HAVE_LIBATSPI
+# include "msd-a11y-keyboard-atspi.h"
+#endif
 #include "msd-a11y-preferences-dialog.h"
 
 #define CONFIG_SCHEMA "org.mate.accessibility-keyboard"
+#define KEY_CAPSLOCK_BEEP_ENABLED "capslock-beep-enable"
```

Comment:
You always use this macro within blocks where HAVE_LIBATSPI  is checked, so to be consistent you would have to declare it based on HAVE_LIBATSPI

Reply:
I think it starts getting a little too many preprocessor conditionals, but indeed it makes sense.

BTW, not that it matters for this one, but there's the same issue with `NOTIFICATION_TIMEOUT` on the next line.